docs/reference/redirects.asciidoc

```diff
@@ -1225,3 +1225,9 @@ See <<size-your-shards>>.
 === elasticsearch-croneval parameters
 
 See <<elasticsearch-croneval-parameters>>.
+
+[role="exclude",id="autoscaling-get-autoscaling-decision"]
+=== Autoscaling decision API
+
+The autoscaling decision API has been renamed to capacity,
+see <<autoscaling-get-autoscaling-capacity>>.
```

Comment:
I'm not sure this is necessary, since we don't publish these pages for released (non-snapshot) versions? That is, today, if you navigate to this page in a snapshot version, it exists, but once we release the docs for that version, the page disappears.

Reply:
I think I need to add this temporarily since it is referenced by the javascript api docs:

https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/api-reference.html#_autoscaling_getautoscalingdecision

Once those have been corrected to the new link, I will circle back to remove this redirect.

Reply:
Okay, thanks for clarifying. I don't think this API should be exposed in the JavaScript client at the moment.